src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -14,37 +14,33 @@
 import edu.wpi.first.math.controller.ArmFeedforward;
 import edu.wpi.first.math.controller.ProfiledPIDController;
 import edu.wpi.first.math.trajectory.TrapezoidProfile;
-import edu.wpi.first.math.util.Units;
 import edu.wpi.first.wpilibj.DutyCycleEncoder;
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants.RobotConstants;
+import frc.robot.parameters.ArmParameters;
 import frc.robot.parameters.ElevatorLevel;
 import frc.robot.parameters.MotorParameters;
 
-public class CoralArm extends SubsystemBase {
-
-  private static final double MASS = 1.361; // mass of the arm in Kg
-  private static final double GEAR_RATIO = 75.0;
-  private static final MotorParameters MOTOR = MotorParameters.KrakenX60;
-
-  private static final double EFFICIENCY = 1.0;
-  private static final double RADIANS_PER_REVOLUTION = 1.0;
-  private static final double MAX_ANGULAR_SPEED =
-      EFFICIENCY * MOTOR.getFreeSpeedRPM() * RADIANS_PER_REVOLUTION / 60.0;
-  private static final double ARM_LENGTH = Units.inchesToMeters(12);
-  private static final double MAX_ANGULAR_ACCELERATION =
-      EFFICIENCY * ((2 * MOTOR.getStallTorque() * GEAR_RATIO) / (MASS * ARM_LENGTH));
-  private static final TrapezoidProfile.Constraints CONSTRAINTS =
-      new TrapezoidProfile.Constraints(MAX_ANGULAR_SPEED * 0.3, MAX_ANGULAR_ACCELERATION * 0.5);
-  private static final double KS = 0.15;
-  private static final double KV = RobotConstants.MAX_BATTERY_VOLTAGE / MAX_ANGULAR_SPEED;
-  private static final double KA = RobotConstants.MAX_BATTERY_VOLTAGE / MAX_ANGULAR_ACCELERATION;
-  private static final double KG = KA * 9.81;
-
-  private final TalonFX motor = new TalonFX(RobotConstants.CAN.TalonFX.ARM_MOTOR_ID);
-  private final DutyCycleEncoder absoluteEncoder =
-      new DutyCycleEncoder(RobotConstants.DigitalIO.ARM_ABSOLUTE_ENCODER);
+public class Arm extends SubsystemBase {
+
+  private static double MASS; // mass of the arm in Kg
```

Comment:
Since the former constant values depend on the arm parameters, they can no longer be `static`; they'll need to be instance variables using lower camel-case naming.